tests/checks/placeholder_pages/placeholder_pages_test.go

```diff
@@ -0,0 +1,238 @@
+//go:build e2e
+// +build e2e
+
+package placeholder_pages
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	. "github.com/kedacore/http-add-on/tests/helper"
+)
+
+const (
+	testName        = "placeholder-pages-test"
+	testServiceName = testName
+	testNamespace   = testName + "-ns"
+)
+
+type placeholderTemplateData struct {
+	TestNamespace   string
+	TestName        string
+	TestServiceName string
+}
+
+const placeholderTemplate = `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{.TestNamespace}}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.TestName}}
+  namespace: {{.TestNamespace}}
+  labels:
+    app: {{.TestName}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{.TestName}}
+  template:
+    metadata:
+      labels:
+        app: {{.TestName}}
+    spec:
+      containers:
+      - name: app
+        image: kennethreitz/httpbin:latest
+        ports:
+        - containerPort: 80
+          name: http
+        resources:
+          requests:
+            memory: "64Mi"
+            cpu: "100m"
+          limits:
+            memory: "128Mi"
+            cpu: "200m"
```

Comment:
let's use the same image as for other e2e tests
```suggestion
      containers:
      - name: {{.TestName}}
        image: registry.k8s.io/e2e-test-images/agnhost:2.45
        args:
        - netexec
        ports:
        - name: http
          containerPort: 8080
          protocol: TCP
        readinessProbe:
          httpGet:
            path: /
            port: http
```